tutorial/sample-problems/Parametric/SpaceCurveGraph.pg

```diff
@@ -13,7 +13,7 @@
 
 #:% name = Space Curve Graph
 #:% type = Sample
-#:% subject = [parametric, graph]
+#:% subject = [parametric, vector calculus]
```

Comment:
Instead of just removing the `graph` subject, make `graph` a category.  This is still a problem with a graph.